Nimble_Snapshots/CurrentTestCaseTracker.swift

```diff
@@ -17,12 +17,8 @@ import XCTest
 
 extension XCTestCase {
     var sanitizedName: String? {
-        guard let fullName = self.name else {
-            return nil
-        }
-
         let characterSet = CharacterSet(charactersIn: "[]+-")
-        let name = fullName.components(separatedBy: characterSet).joined()
+        let name = self.name.components(separatedBy: characterSet).joined()
```

Comment:
@marcgovi I updated the project to use XCode 9 beta 6 and `self.name` is still non-optional, 😥. Please let me know if you can point me where did you saw it?? Thanks